tests/pages/firefox/accounts.py

```diff
@@ -2,24 +2,35 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from selenium.common.exceptions import TimeoutException
 from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as expected
+from selenium.webdriver.support.ui import WebDriverWait as Wait
 
 from pages.firefox.base import FirefoxBasePage
-from pages.regions.download_button import DownloadButton
```

Comment:
We should always try and use the download button page region, as this asserts that there's only one download link displayed within the button. Without this, the download button CSS could be busted and the test could still pass.